src/platform/windows/nvprefs/driver_settings.cpp

```diff
@@ -175,6 +170,55 @@ namespace nvprefs {
       return false;
     }
 
+    if (get_nvprefs_options().force_disable_vsync)
+    {
+      NVDRS_SETTING setting = {};
+      setting.version = NVDRS_SETTING_VER;
+      status = NvAPI_DRS_GetSetting(session_handle, profile_handle, VSYNCMODE_ID, &setting);
+      if (status != NVAPI_OK) {
+        nvapi_error_message(status);
+        error_message("NvAPI_DRS_GetSetting() VSYNCMODE failed");
+        return false;
+      }
+
+      // TODO: save current setting and restore it later
+      setting.u32CurrentValue = VSYNCMODE_FORCEOFF;
+
+      status = NvAPI_DRS_SetSetting(session_handle, profile_handle, &setting);
+      if (status != NVAPI_OK) {
+        nvapi_error_message(status);
+        error_message("NvAPI_DRS_SetSetting() VSYNCMODE failed");
+        return false;
+      }
+    }
+
+    if (get_nvprefs_options().enable_frame_limiter)
+    {
+      NVDRS_SETTING setting = {};
+      setting.version = NVDRS_SETTING_VER;
+      status = NvAPI_DRS_GetSetting(session_handle, profile_handle, FRL_FPS_ID, &setting);
+      if (status != NVAPI_OK) {
+        nvapi_error_message(status);
+        error_message("NvAPI_DRS_GetSetting() FRL_FPS_ID failed");
+        return false;
+      }
+
+      // TODO: save current setting and restore it later
+      setting.u32CurrentValue = 60; // TODO: change this according to host refresh rate
```

Comment:
I suppose you can get this from https://github.com/LizardByte/Sunshine/pull/2894 after it's merged.